content/kotlin/concepts/datetime/terms/Month/Month.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: '.Month'
+Description: 'Retrieves the month from a Kotlin DateTime object as an integer.'
+Subjects:
+  - 'Kotlin'
+  - 'DateTime'
+Tags:\
+  - 'Date'
+  - 'Methods'
+  - 'Kotlin'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+
+The **`.Month()`** method in Kotlin is used to retrieve the month from a given date object. This method is essential when working with `date` and `time` in Kotlin, allowing developers to extract and manipulate the month component of a date. It's particularly useful in scenarios where the month needs to be displayed, compared, or used in calculations. In Kotlin's date-time API, the month is typically represented as an integer, where `1` corresponds to `January` and `12` to `December`.
+
+## Syntax
+
+```kotlin
+val month: Int = date.Month()
+```
+
+- *date*: An instance of a date object from which you want to extract the month.
+- *month*: This is an integer representing the month extracted from the date object. Typically, in Kotlin, months are represented as integers ranging from 1 (January) to 12 (December).
+
+## Example
+
+```kotlin
+import java.time.LocalDate
+
+fun main() {
+    val currentDate = LocalDate.now() 
+    val month = currentDate.monthValue
+
+    println("The current month is: $month")
+}
+```
+
+In this example, `LocalDate.now()` is used to get the current date. The `.monthValue` property (which serves a similar purpose to a `.Month()` method) of the `LocalDate` object is then called to retrieve the month from the current date. The month is then printed to the console. Assuming the current date is November 1, 2023, the output will be `11`.
```

Comment:
Remove this sentence from here and move it above the example code snippet as an introductory sentence. Also, remove the last output part and add the output in its own block. Like this:
```shell
then your output

Reply:
<img width="114" alt="image" src="https://github.com/Codecademy/docs/assets/94860622/ab094437-20ac-4719-affe-36b5f2d5c044">
This is how to make output block 👆🏻